cmd/cluster-init/main.go

```diff
@@ -42,9 +42,14 @@ func validateOptions(o options) []error {
 		}
 	}
 	if o.releaseRepo == "" {
-		errs = append(errs, errors.New("--release-repo must be provided"))
+		//If the release repo is missing, further checks won't be possible
+		return append(errs, errors.New("--release-repo must be provided"))
 	}
 	if o.clusterName != "" {
+		existsFor, err := periodicExistsFor(o)
+		if existsFor || err != nil {
+			errs = append(errs, fmt.Errorf("cluster: %s already exists", o.clusterName))
```

Comment:
you eat the `err` here

Reply:
Good point. I suppose we shouldn't just assume that the error could only mean an existing periodic.